_tutorials/convert-html-site-to-jekyll.md

```diff
@@ -0,0 +1,362 @@
+    </li>
+{% endfor %}
+</ul>{% endraw %}
+```
+
+Open `index.md` in your root directory and specify the `home` layout:
+
+```
+---
+title: My Awesome Site
+layout: home
+---
+```
+
+When a layout specifies another layout, it means the content of this layout will be stuffed into the `{% raw %}{{ content }}{% endraw %}` tag of the layout specified in the front matter. In this case, the content from `index.md` will be pushed into the `{% raw %}{{ content }}{% endraw %}` tags in the `home` layout. Then the `home` layout will be pushed into the `{% raw %}{{ content }}{% endraw %}` tags of the `default` layout.
+
+(If you don't specify a layout, `default` is used by default. Specifying it in the front matter tags is optional.)
+
+By the way, the we've only scratched the surface of what you can do with `for` loops in retrieving posts. For example, if you want to display posts from a specific category, you can do so by adding a `categories` property to your post's front matter, and then looking in those categories. Further, you can limit the number of results by adding a `limit` property. Here's an example:
+
+```
+{% raw %}<ul class="myposts">
+{% for post in site.categories.podcasts limit:3 %}
+    <li><a href="{{ post.url }}">{{ post.title}}</a>
+    <span class="postDate">{{ post.date | date: "%b %-d, %Y" }}</span>
+    </li>
+{% endfor %}{% endraw %}
+```
+
+This loop would get the latest 3 posts that have a category called `podcasts` in the front matter.
+
+## 9. Configure navigation
+
+Now that you've configured posts, let's configure page navigation. Most websites have some navigation either in the sidebar or header area.  
+
+In this tutorial, we'll assume you've got a simple list of pages you want to generate. You can use Jekyll's `for` loop to iterate through a list of pages that you maintain in a separate data file.
+
+Create a folder in your Jekyll project called `_data`. In it, create a file called `sidebar_links.yml` with this content:
+
+```
+- title: Sample page 1
+  url: /sample1/
+
+- title: Sample page 2
+  url: /sample2/
+
+- title: Sample page 3
+  url: /sample3/
+```
+
+(You can store additional properties for each item in this YAML file as desired .)
+
+Identify the part of your code where the list of pages appears. Usually this is a `<ul>` element with various child `<li>` elements. Replace the code with the following:
+
+```
+{% raw %}<ul>
+    {% for p in site.data.sidebar_links %}
+    <li><a href="{{ p.url }}">{{ p.title }}</a></li>
+    {% endfor %}
+</ul>{% endraw %}
+```
+
+If you have more sophisticated requirements around navigation, see the [detailed tutorial on navigation]({% link _tutorials/navigation.md %}). 
+
+
+## 10. RSS feed
+
+Your Jekyll site needs an RSS feed. Here's the [basic RSS feed syntax](http://www.w3schools.com/xml/xml_rss.asp). To create an RSS file in Jekyll, create a file called `feed.xml` and add the following:
+
+```xml
+---
+layout: null
```

Comment:
Actually, when I use `nil` instead of `null`, Jekyll complains in the command line about not being able to find a layout with the name of `nil`. But it doesn't complain when it looks for `null`.